infrastructure/output.tf

```diff
@@ -1,9 +1,9 @@
 output "vaultName" {
-  value = module.civil_sdt_key_vault.key_vault_name
+  value = data.azurerm_key_vault.key_vault.name
 }
 
 output "vaultUri" {
-  value = module.civil_sdt_key_vault.key_vault_uri
+  value = data.azurerm_key_vault.key_vault.vault_uri
```

Comment:
Change value to `module.civil_sdt_key_vault.key_vault_id` as this needs to reference the civil-sdt key vault.